cmake/winml.cmake

```diff
@@ -475,6 +493,82 @@ if (onnxruntime_USE_DML)
   target_add_dml(winml_lib_api)
 endif(onnxruntime_USE_DML)
 
+
+###########################
+# Add winml_lib_api_experimental_dir
+###########################
+
+# Add static library that will be archived/linked for both static/dynamic library
+add_library(winml_lib_api_experimental STATIC
+  ${winml_lib_api_experimental_dir}/Dummy.cpp
```

Comment:
These will be replaced in the future as new winrt features are added right?

Reply:
yes, we wont be shipping Dummy as part of the api ;) it will be removed - this was for my local tests to pass.